config/v1.6/aws-k8s-cni-cn.yaml

```diff
@@ -77,6 +77,7 @@ spec:
                     operator: In
                     values:
                       - amd64
+                      - arm64  
```

Comment:
Is the arm64 support intended for 1.6.4?

Reply:
Yeah now the image is built as a manifest list of both arch types :)